service/common/error_handlers.py

```diff
@@ -71,19 +71,19 @@ def method_not_supported(error):
     )
 
 
-@app.errorhandler(status.HTTP_409_CONFLICT)
-def resource_conflict(error):
-    """Handles resource conflicts with HTTP_409_CONFLICT"""
-    message = str(error)
-    app.logger.warning(message)
-    return (
-        jsonify(
-            status=status.HTTP_409_CONFLICT,
-            error="Conflict",
-            message=message,
-        ),
-        status.HTTP_409_CONFLICT,
-    )
+# @app.errorhandler(status.HTTP_409_CONFLICT)
+# def resource_conflict(error):
+#     """Handles resource conflicts with HTTP_409_CONFLICT"""
+#     message = str(error)
+#     app.logger.warning(message)
+#     return (
+#         jsonify(
+#             status=status.HTTP_409_CONFLICT,
+#             error="Conflict",
+#             message=message,
+#         ),
+#         status.HTTP_409_CONFLICT,
+#     )
```

Comment:
looks good if we're not using it.

Reply:
yeah, i don't think we will. we can always uncomment it in the further iterations if needed.